lib/bigfiles.rb

```diff
@@ -48,11 +50,22 @@ def add_help_option(opts, options)
       end
     end
 
+    def add_num_files_option(opts, options)
+      opts.on('-n', '--num-files number-here',
+              Integer,
+              'Top number of files to show--' \
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.